packages/@lwc/engine-core/src/framework/wiring.ts

```diff
@@ -339,11 +340,14 @@ export function installWireAdapters(vm: VM) {
             const hasDynamicParams = wireDef.dynamic.length > 0;
             ArrayPush.call(wiredConnecting, () => {
                 connector.connect();
-                if (hasDynamicParams) {
-                    Promise.resolve().then(computeConfigAndUpdate);
-                } else {
-                    computeConfigAndUpdate();
+                if (featureFlags.ENABLE_WIRE_DEFERRED_FIRST_UPDATE) {
```

Comment:
The feature flags only allow [one member expression in the test](https://github.com/salesforce/lwc/blob/master/packages/%40lwc/features/src/babel-plugin/index.js#L65-L74).